import "EquipmentInfo.proto";

message DseAllEquipmentInfo
{
	repeated EquipmentInfo		infos = 1;
}